unfeature old okr post, upload 2023/24 goals (#548)
chihacknight/chihacknight.org: _posts/blog/2023-07-01-2023-retreat-okrs.md

layout: blog
categories: 
  - blog
title: "Chi Hack Night | 2023-2024 Goals"
description: ""
date: 2023-08-22
image: /images/blog/2023-board-retreat.jpg
author: Sean Watland
author_url: https://chihacknight.org/board-of-directors.html
author_image: /images/people/board/sean_watland.jpg
author_bio: 
author2: 
author2_url: 
author2_image: 
author2_bio: 
published: true
featured: true
---
<p class="text-center"><img src="/images/blog/2023-board-retreat.jpg" alt="Picture of our Board of Directions from our board retreat" class='img-responsive' style='width:100%;'/>
</p>

> On June 3th, 2023, the Board of Directors held our first in-person *board retreat*! This retreat served as an excellent opportunity for the 10 Board of Directors to have more focused discussions around what membership means at Chi Hack Night, plan our upcoming **OKRs** for the year, and enjoy copious amounts of bagels & coffee. 

Reviewer: ```suggestion
On June 3rd, 2023, the Board of Directors held our first in-person *board retreat*! This retreat served as an excellent opportunity for the 10 Board of Directors to have more focused discussions around what membership means at Chi Hack Night, plan our upcoming **OKRs** for the year, and enjoy copious amounts of bagels & coffee. 
```

Our retreat was held from 10 am until 3 pm, downtown Chicago at [dscout](www.dscout.com) corporate headquarters. (Thank you [dscout](www.dscout.com) for graciously hosting us for the day!)

> In planning our goals for the upcoming year - three larger goals out of our discussions: 

Reviewer: ```suggestion
Through our discussions, the board identified three primary goals for the upcoming year: 
```

1. Continuing to run a high quality civic tech event

> 2. Align our membership experiences with our hybrid event

Reviewer: ```suggestion
2. Align our membership requirements with our hybrid event format
```

3. Expand the broader awareness of Chi Hack Night

For the third year, we've used an **OKR** framework to plan and measure our goals. **OKRs** (short for **Objective** & **Key Results**) help us know: 
1. What are our important focus areas - the **Objectives** 
2. How do we know we've succeeded - the **Key Results**

*If you are curious to learn more about OKRs and how they work, here's an [article](https://www.whatmatters.com/faqs/okr-meaning-definition-example) that shares more about how they work.*

> While you'll see and hear more about the specifics on various ways of going about this, we wanted to share these out so that our wider community can see what we've determined to focus on for the coming year. 

Reviewer: ```suggestion
While you'll see and hear more about the specifics as we work on these throughout the year, we wanted to share these out now so that our wider community can see what we're planning to focus on in the coming months. 
```

# **2023-2024 OKRs**
*from 7/1/23 - 6/30/24*

## **Objective 1 - Run the event** {#objective-1-run-the-event}
_Produce a live, weekly civic engagement event that meaningfully includes diverse speakers and attendees_
* **KR 1** - Create a biannual survey for Chi Hack Night community asking people about belonging / welcomingness of events & quality and representativeness of presenters
* **KR 2** - Book at least three presentations from each of the following groups: government, journalism, academia, nonprofit, and advocacy

> * **KR 3** - 65% or higher response rate from presenters on our demographic survey.  

Reviewer: ```suggestion
* **KR 3** - 65% or higher response rate from presenters on our demographic survey
```

## **Objective 2 - Increase Membership Engagement** {#objective-2-increase-membership-engagement}

> * **KR 1** - 20% of volunteer roles available are filled by members (either current or future members)

Reviewer: Is this non-board member members?

Author: yes - non-board members. I intentionally did not include that piece for brevity - cuz then I felt like I would need to explain why that stipulation is needed. So I just opted to keep it short.
